docs/source/howto.rst

```diff
@@ -0,0 +1,98 @@
+.. _how-to:
+
+#############
+How-To Guides
+#############
+
+.. _how-to:install_archive:
+
+Installing from archive or folder
+=================================
+
+In case the automated install commands fail, or the pseudopotential family you want to use is not supported, you can install the pseudopotential family manually with the following command:
+
+.. code-block:: console
+
+    $ aiida-pseudo install family <ARCHIVE_OR_FOLDER> <LABEL>
+
+where ``<ARCHIVE_OR_FOLDER>`` should be replaced with the archive or folder that contains the pseudopotential files and ``<LABEL>`` with the label to give to the family.
+
+.. important::
+
+    In order to properly parse the elements from the pseudopotential files, they must adhere ``ELEMENT.EXTENSION`` filename format.
+    There must also not be any extraneous files in the folder.
```

Comment:
```suggestion
    The unpacked archive should also contain nothing else besides the pseudopotential files (i.e. no other files or folders) and the pseudopotentials should be present at the top level (not nested in subfolders)..
```
Feel free to rephrase but this seems important additional information on expected format